src/components/Footer.astro

```diff
@@ -16,6 +16,22 @@ import maplibreLogo from "../../public/img/maplibre-logo.svg";
 
       <div class="col-8">
         <div class="nav justify-content-end">
+          <a
+            class="nav-link text-white"
+            href={new URL("rss.xml", Astro.site)}
+            aria-label="MapLibre RSS Feed"
+          >
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              viewBox="0 0 16 16"
+              width="16"
+              height="16"
+              ><path
+                fill="#fff"
+                d="M2.002 2.725a.75.75 0 0 1 .797-.699C8.79 2.42 13.58 7.21 13.974 13.201a.75.75 0 0 1-1.497.098 10.502 10.502 0 0 0-9.776-9.776.747.747 0 0 1-.7-.798ZM2.84 7.05h-.002a7.002 7.002 0 0 1 6.113 6.111.75.75 0 0 1-1.49.178 5.503 5.503 0 0 0-4.8-4.8.75.75 0 0 1 .179-1.489ZM2 13a1 1 0 1 1 2 0 1 1 0 0 1-2 0Z"
```

Comment:
is the SVG duplicated? Can it be stored in a css file or something?

Reply:
yeah this icon was duplicated, other icons on the footer didn't have a file so I didn't want to break the consistency. i'll raise another PR moving them too to SVG.

Reply:
I raised https://github.com/maplibre/maplibre.github.io/pull/380 which does the same pattern for the other footer icons.

Reply:
I've merged the other PR, thanks! 
I'll leave this one to someone else...